care/facility/api/serializers/patient.py

```diff
@@ -104,6 +111,13 @@ def validate_facility(self, value):
             raise serializers.ValidationError("facility not found")
         return value
 
+    def validate_countries_travelled(self, value):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5380979)